clustered/integration-test/src/test/java/org/ehcache/clustered/management/AbstractClusteringManagementTest.java

```diff
@@ -194,16 +193,28 @@ public static void afterClass() throws Exception {
     if (cacheManager != null && cacheManager.getStatus() == Status.AVAILABLE) {
 
       if (nmsService != null) {
-        Context ehcacheClient = readTopology().getClient(ehcacheClientIdentifier).get().getContext().with("cacheManagerName", "my-super-cache-manager");
-        nmsService.stopStatisticCollector(ehcacheClient).waitForReturn();
+        readTopology().getClient(ehcacheClientIdentifier)
```

Comment:
What's this about?

Reply:
This was failing because of an earlier failure in the test. So it was hiding the real problem in the build failed we experienced.